src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -334,15 +334,33 @@ export const TimelineNav = ({
   };
 
   /**
-   * onClick function
-   * 1) Triggered by 'Back' button on < lg viewports
-   * 2) Return focus to the button that was clicked to open the current panel; activeIndexState holds the index of the last nav item selected in the timelinNavItemRefs array
+   * onBack (used by 'Back' button on < lg viewports)
+   * 1) Return focus to the button that was clicked to open the current panel; activeIndexState holds the index of the last nav item selected in the timelinNavItemRefs array
+   * 2) Because the 'Back' button's onFocus listener removes the selected nav item from the tab order, we need to manually insert it back into the tab order by setting its tabIndex back to "0"
    * 3) Body panel is visible/hidden depending on true/false value of isActive; hide it by setting isActive to false
    */
-  const onClick = () => {
-    timelineNavItemRefs[activeIndexState].current?.focus(); /* 2 */
+  const onBack = () => {
```

Comment:
Changing this to `onBack()` to prevent confusion of which element it belongs to.